CoWork_klasy.py

```diff
@@ -0,0 +1,132 @@
+class Desks:
+    def __init__(self, name, desk_type, price, status):
+        self.name = name
+        self.desk_type = desk_type
+        self.price = price
+        self.status = status
+
+    def __str__(self):
+        return f" {self.desk_type}, Cena: {self.price} PLN/h, Status: {self.status}"
+
+
+
+# Create instances for each entry in the dictionary
+desks_instances = {
```

Comment:
To tutaj tak narpawde jest zhardkodowane, powinniscie miec funkcje która otwiera i wczytuje dane z JSONA, a potem tworzy isntajce tej klasy na podstawie danych z JSONA